<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>561</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <widget class="QRadioButton" name="hybrid_rb">
       <property name="text">
        <string>Гибрид</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="place_input"/>
       </item>
       <item>
        <widget class="QPushButton" name="find_b">
         <property name="text">
          <string>Искать</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clear_b">
         <property name="text">
          <string>Очистить</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="satellite_rb">
       <property name="text">
        <string>Спутник</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QRadioButton" name="scheme_rb">
       <property name="text">
        <string>Схема</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="3">
      <widget class="QGraphicsView" name="display"/>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="QLabel" name="adress_d">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
